src/ButtonPage.scss

```diff
@@ -0,0 +1,9 @@
+@import "@material/button/dist/mdc.button";
+
+.mdc-button {
```

Comment:
Preferably add a custom class to the buttons to avoid overriding mdc classes directly.